main/signers/lattice/Lattice/index.ts

```diff
@@ -65,7 +61,7 @@ export default class Lattice extends Signer {
   accountLimit = 5
   tag = ''
 
-  constructor (deviceId: string, name: string, tag: string) {
+  constructor(deviceId: string, name: string, tag: string) {
```

Comment:
removed a space here that's consistent with our formatting standards

Reply:
[Commit](https://github.com/floating/frame/pull/845/commits/ebc0fe38035651a7cd9ef17cff9b9b132f1ea21b)